src/main/java/com/worksap/nlp/elasticsearch/plugins/chikkar/DictionaryManager.java

```
@@ -342,7 +346,7 @@ public synchronized void addDictionary(RelationManager relationMgn, String dictP
                 }
             });
         } catch (IOException e) {
-            System.err.println(e);
+            log.error(e.getMessage());
```

In this case, the user cannot know an error is happen. It is better to throw an exception, isn't it?

---

Yes, you are right. Modified.